internal/consensus/reactor.go

```diff
@@ -687,6 +691,20 @@ OUTER_LOOP:
 				"height", prs.Height,
 				"vote", vote,
 			)
+		} else {
+			if c := sendEntireCommit(logger, conR.conS, rs, ps, prs, rng); c != nil {
+				commit, ok := (*c).(*types.Commit)
+				if !ok {
+					panic("Wrong type,got")
+				}
```

Comment:
```suggestion
				commit := (*c).(*types.Commit)
```
If you don't use the "ok" it'll panic anyway